src/application/CMakeLists.txt

```diff
@@ -14,9 +14,15 @@ endif()
 
 find_package(DevIL COMPONENTS IL ILU) # yields IL_FOUND, IL_LIBRARIES, IL_INCLUDE_DIR
 
-set(Boost_INCLUDE_DIRS "")
-set(Boost_LIBRARIES "")
-find_package(Boost 1.53.0 REQUIRED COMPONENTS filesystem program_options)
+if(PopSift_BOOST_USE_STATIC_LIBS)
+  set(Boost_USE_STATIC_LIBS ON)
+endif()
+find_package(Boost 1.53.0 REQUIRED COMPONENTS filesystem program_options system)
+if(WIN32)
+  add_definitions("-DBOOST_ALL_NO_LIB")
+  link_directories(Boost_LIBRARRY_DIR_DEBUG)
+  link_directories(Boost_LIBRARRY_DIR_RELEASE)
```

Comment:
normally you should not need these because it is taken care of by the later `target_link_libraries` and `target_compile_definitions`.
Did you find it necessary in your tests?

Reply:
Those lines were move from the root `CMakeLists.txt`. My understanding matches yours. I expect the `${Boost_LIBRARIES}` to be full paths, making these lines obsolete.

Unfortunately, I don't have access to a WIN32 machine, so I can't verify whether or not they're still required. Happy to remove them if requested...

Reply:
yes you can remove that, thanks

Reply:
Removed.